examples/temporal/README.md

```diff
@@ -0,0 +1,97 @@
+# Running SkyPilot Tasks in Temporal Workflows
+
+This example demonstrates how to launch SkyPilot tasks and manage them in a Temporal workflow. 
+
+<p align="center">
+  <img src="https://i.imgur.com/rxlO2pJ.png" width="800">
+</p>
+
+## Prerequisites
+
+- [SkyPilot](https://docs.skypilot.co/en/latest/getting-started/installation.html) installed on temporal workers
+- [Remote SkyPilot API server](https://docs.skypilot.co/en/latest/reference/api-server/api-server.html)
+
+Using the SkyPilot API Server eliminates the need for managing SkyPilot state across workers and allows credentials to be stored securely in the API server.
```

Comment:
This feels too much info in this section. No explanation needed at this stage, I feel?